calendar/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+  - abhi-g
```

Comment:
Only list the people that have explicitly ack'd being calendar admins. Based on #342 that would just be you.

Reply:
Also jfogarty and yuzliu (see https://github.com/kubeflow/community/issues/342)

The others were in the root OWNERS file, should I remove them fro here?

Reply:
Now also my colleague nicholas-abad

Reply:
Yes. Only list the folks that are specifically helping out with the calendar.
Lets cap it at the 4 people you mentioned.

Please alphabetize

Reply:
@karlschriek please remove anyone who isn't already a member of the Kubeflow GitHub org.